src/components/Disputes/Import.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable  */
+import React, { useState, useEffect } from 'react'
+import { SidePanel, Button, DropDown, TextInput, textStyle } from '@aragon/ui'
+import { Octokit } from '@octokit/rest'
+import { useWallet } from 'use-wallet'
+
+function Import() {
+  const wallet = useWallet()
+
+  const [USERNAME, SETUSERNAME] = useState(null)
+
+  const [opened, setOpened] = useState(false)
+
+  const [list, setList] = useState([])
+
+  const TOKEN = localStorage.getItem('ACCESS TOKEN')
+
+  const octokit = new Octokit({
+    auth: TOKEN,
+  })
+
+  const [publicRepos, setPublicRepos] = useState(0)
+  async function userInfo() {
+    const { data } = await octokit.request('/user')
+    console.log('userinfo', data)
+    let num = data.public_repos
+    setPublicRepos(num - 1)
+    return data
+  }
+  async function listUserRepos() {
+    console.log(localStorage.getItem('ACCESS TOKEN'))
+    setOpened(true)
+
+    let repoList = []
+    const user = await userInfo()
+    for (let page = 1; page <= Math.ceil(user.public_repos / 100); page++) {
+      const { data } = await octokit.repos.listForAuthenticatedUser({
+        page: page,
```

Comment:
Since you have hardcoded the `per_page` with 100, you need to pass `per_page: 100` as the attribute here. Check the Octokit example. Otherwise it will only fetch 30 repos at a time.

Reply:
Fixed